src/pages/index.js

```diff
@@ -198,6 +301,15 @@ function closeModal(modal) {
   modal.classList.remove("modal_is-opened");
   modal.removeEventListener("click", overlayClickListener);
   document.removeEventListener("keydown", handleEscape);
+  if (modal === deleteModal) {
+    deleteModalForm.reset();
+    selectedCard = null;
+    selectedCardId = null;
+  }
```

Comment:
Please, delete it. You don't need it in the universal function

There should **not** be any code in universal functions that doesn’t suit every popup.

Reply:
This hasn't been corrected